adcm_aio_client/core/objects/cm.py

```diff
@@ -127,3 +128,19 @@ def get_own_path(self: Self) -> Endpoint:
 
 class ComponentsNode(NonPaginatedChildAccessor[Service, Component, None]):
     class_type = Component
+
+
+class ADCM(InteractiveObject, WithActions, WithConfig):
+    @cached_property
+    async def version(self: Self) -> str:
+        root_requester = DefaultRequester(root_path="", base_url="")
+        root_requester.api_root = getattr(self._requester, "api_root")
+        response = await root_requester.get("versions")
+        return response.as_dict()["adcm"]["version"]
```

Comment:
requester is dependency, it can't be instantiated in here

probably we need a way to override `root_path` for a time being?

@DanBalalan maybe you have ideas how to do this?